cdec/installation-manager-core/src/main/java/com/codenvy/im/commands/CommandLibrary.java

```diff
@@ -80,7 +80,7 @@ public static Command createReplaceCommand(String file, String replacingToken, S
     public static Command createReplaceCommand(String file, String replacingToken, String replacement, boolean withSudo) {
         String cmd = format("sudo cat %3$s | sed ':a;N;$!ba;s/\\n/~n/g' | sed 's|%1$s|%2$2s|g' | sed 's|~n|\\n|g' > tmp.tmp && sudo mv tmp.tmp %3$s",
                             replacingToken,
-                            replacement.replace("\\$", "\\\\$").replace("\n", "\\n"),
+                            replacement.replace("\\$", "\\\\$").replace("\n", "\\n").replaceAll("&", "\\\\&"), // http://sed.sourceforge.net/sedfaq3.html: "To enter a literal ampersand on the RHS, type '\&'."
```

Comment:
Yes, we need it because, as it was described in description of target issue CODENVY-543, there could be value with a number of "&" symbols and all of them must be replaces on "\&" to avoid breaking of "sed" command.